src/main/java/frc/robot/RobotContainer.java

```diff
@@ -321,6 +321,14 @@ public void seedIntakeEncoder() {
     intake.seedEncoder();
   }
 
+  public void seedAlgeaEncoder() {
+    algeaRemover.seedEncoder();
+  }
+
+  public void seedClimberEncoder() {
+    climber.seedEncoder();
+  }
+
```

Comment:
Could you combine these methods into one seedEncoders() method that does all three?

Reply:
Delete the individual methods now that we have one that does all of them